docs/variants/novacustom_ns5x_tgl/hardware-matrix.md

```diff
@@ -79,11 +81,13 @@ for the NS51/70 platforms.
 | **Wireless card**              | Intel Wi-Fi 6 AX201                              |
 | **Display**                    | Display 1: HDMI 1920x1080p                       |
 | **Network**                    | Local network wired connection                   |
-| **Internal devices**           | 1. 1920x1080 14 inch screen                      |
+| **Internal devices**           | 1. 1920x1080 17 inch screen                      |
 |                                | 2. Internal keyboard with LED backlight          |
 |                                | 3. Touchpad                                      |
 |                                | 4. Camera                                        |
 |                                | 5. Audio subsystem                               |
 | **Attached devices**           | Thunderbolt 4 Port: Wavlink 100W PD Pro docking station |
```

Comment:
a/a, but also, TGL devices will not be tested with that dock, but with the displaylink dock. Exact dock models and versions are listed in https://github.com/Dasharo/docs/pull/715

Reply:
Done: 734da51b6b6304d1830d9581a9927a0d322dc275